src/panels/profile/ha-set-suspend-row.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { fireEvent, HASSDomEvent } from "../../common/dom/fire_event";
+import "../../components/ha-switch";
+import type { HaSwitch } from "../../components/ha-switch";
+import type { HomeAssistant } from "../../types";
+import "./ha-settings-row";
+
+declare global {
+  // for fire event
+  interface HASSDomEvents {
+    "hass-suspend-when-hidden": { suspend: HomeAssistant["suspendWhenHidden"] };
+  }
+  // for add event listener
+  interface HTMLElementEventMap {
+    "hass-suspend-when-hidden": HASSDomEvent<{
+      suspend: HomeAssistant["suspendWhenHidden"];
+    }>;
+  }
+}
+
+@customElement("ha-set-suspend-row")
+class HaSetSuspendRow extends LitElement {
+  @property() public hass!: HomeAssistant;
+
+  @property() public narrow!: boolean;
+
+  protected render(): TemplateResult {
+    return html`
+      <ha-settings-row .narrow=${this.narrow}>
+        <span slot="heading">
+          ${this.hass.localize("ui.panel.profile.suspend.header")}
+        </span>
+        <span slot="description">
+          ${this.hass.localize("ui.panel.profile.suspend.description")}
+        </span>
+        <ha-switch
+          .checked=${this.hass.suspendWhenHidden}
```

Comment:
Should this be comparing against false since default is true?
```suggestion
          .checked=${this.hass.suspendWhenHidden !== false}
```

Reply:
Oh nm, we fix that in `connection-mixin.ts`